solution.txt

```diff
@@ -204,12 +214,16 @@ Exercise 15:
 
 // Complete Exercise 15 below...
 
-let total = 0;
-
-numArrays.flat(3).forEach(bigSum);
+var total=0
+for (let i=0; i<numArrays.length; i++) //goes through each iteration of the original array
+  {
+    x=numArrays[i]
+    for (let i=0; i<x.length; i++) //goes through each iteration of each nested array
+    {
+      var y=total //old total before adding new iteration value
+      total=x[i]+y //new total replacing y-var
+    }
 
-function bigSum(n) {
-  total += n;
-}
+  }
```

Comment:
nice! in general, when you have nested `for` loops, you can use `for ( let i = 0; i < something, i++)` for the outside loop, and `for ( let j = 0; j < something, j++)` for the inside loop. this allows you to prevent any confusion that might happen from using the `i` in the inside loop when it's been originally defined for the outside loop